### PR TITLE
fix error

### PR DESCRIPTION
**Pull request by zhaoyansheng163**
import failed: {"message":"Authentication failed: Clone: exit status 128 - remote: HTTP Basic: Access denied\nfatal: Authentication failed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +30,51 @@ source migration-env/bin/activate
 python3 -m pip install -r requirements.txt
 ```
 
-Then start the migration script `python3 migrate.py`.
\ No newline at end of file
+Then start the migration script `python3 migrate.py`.
```
Comment (h44z) [inline README.md:33]: please undo the changes in the readme - the change to migrate.py is sufficient

**Review by h44z — CHANGES_REQUESTED** [review]
please remove the changes to README.md

File: README.md (hunk 1 above)
Comment (zhaoyansheng163) [reply README.md]: I have made the necessary modifications

**Comment by h44z** [comment]
Thanks!
